src/errors.h

```diff
@@ -22,10 +22,11 @@ enum {
 	ERR_INVALID_REQUEST_PARAMETERS = 0x6E05,
 	// Request is not valid in the context of previous calls
 	ERR_INVALID_STATE              = 0x6E06,
-	// Some part of request data is invalid (includes not enough data)
+	// Some part of request data is invalid (or unknown to this app)
+	// (includes not enough data and too much data)
 	ERR_INVALID_DATA               = 0x6E07,
-	// BIP44 path is rejected
-	ERR_INVALID_BIP44_PATH         = 0x6E08,
+	// previously used for rejected BIP44 paths, now we use ERR_REJECTED_BY_POLICY
+	// ERR_INVALID_BIP44_PATH         = 0x6E08,
```

Comment:
From the comment I assume you intend to leave these in the code. What purpose do they serve?

Reply:
I'd prefer not to reuse the error code (e.g. 0x6E08) in the future (to avoid potential confusion if someone has an enum for these codes etc.).